tasksCalendar/view.js

```diff
@@ -141,16 +146,22 @@ function getTasks(date) {
 function setTask(obj, type) {
 	var noteColor = getColor(obj);
 	var noteIcon = getIcon(obj);
+	var textColor = getTextColor(obj);
 	var taskText = obj.text.replace("'", "&apos;");
 	var taskPath = obj.link.path.replace("'", "&apos;");
 	var taskSubpath = obj.header.subpath;
 	var taskLine = taskSubpath ? taskPath+"#"+taskSubpath : taskPath;
 	var style = "";
+	console.log(textColor);
 	// if (noteColor) { style = "color:" + noteColor + ";background:" + noteColor + transparency };
-	if (noteColor) {
-		style = "--task-background:"+noteColor+"33;--task-color:"+noteColor;
+	if (noteColor && textColor) {
+		style = "--task-background:"+noteColor+"33;--task-color:"+noteColor+";--task-text-color:"+textColor;
+	} else if (noteColor && textColor == ""){
```

Comment:
- Second if statement: Problematic declaration `noteColor && textColor == ""`. Here you can use a simple `noteColor && !textColor`